Livecheckables/paperkey.rb

```diff
@@ -1,4 +1,4 @@
 class Paperkey
-  livecheck :url   => "http://www.jabberwocky.com/software/paperkey",
+  livecheck :url   => "https://www.jabberwocky.com/software/paperkey",
```

Comment:
```suggestion
  livecheck :url   => "https://www.jabberwocky.com/software/paperkey/",
```

This redirects to the same URL with a trailing `/`, so we should update the URL to avoid the redirection.